### ISSUE TITLE
Support Shapefile .prj file

### ISSUE THREAD

**Issue by stevage** [perliedman/reproject#36]
I'm interested to know what it would take to support reading Shapefile's .prj files. I had proposed this at [shp2json](http://github.com/mbostock/shapefile/), but maybe it belongs here.

Then one could do:

```
shp2json foo.shp | reproject --prj foo.prj > foo.geojson
```

**Comment by perliedman** [comment]
From my understanding, a Shape file's `.prj` will contain a WKT definition of the projection. Also, it looks like [Proj4.js can parse WKT](https://github.com/proj4js/wkt-parser), although I have never tried it. So, I think a great solution would be to make it possible to supply files as definitions of from/to CRS, instead of just accepting EPSG codes.

I would envision something like adding `--from-crs-file` and `--to-crs-file` to handle this. Does that make sense to you?

**Comment by perliedman** [comment]
Thinking about this further, I added a small change that allows `--from` and `--to` to not only be CRS identifiers, but also actual CRS definitions.

I think this should solve your use case by using it as follows:

```sh
shp2json foo.shp | reproject --from $(foo.prj) --to=epsg:4326 --use-epsg-io > foo.geojson
```

I haven't tested _exactly_ this, since `shp2json` doesn't work on the Windows machine I'm currently on, but that should be a detail.

Give it a spin and see if it works for you!

**Comment by stevage** [comment]
Thanks, that does work. IMHO it would be simpler, more intuitive to have a command line option that directly supports a filename, rather than relying on the `cat` workaround.

**Comment by perliedman** [comment]
Agreed, that might be good. So maybe support `--from-crs-file` and `--to-crs-file` anyway, but let them rely on the same code.
